scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -4,30 +4,33 @@
 # Copyright 2022 NXP
 # SPDX-License-Identifier: Apache-2.0
 
+import logging
+import multiprocessing
 import os
-import shutil
-import re
-import sys
-import subprocess
 import pickle
-import logging
```

Comment:
please, when do this type of change (sorting), do that in a separate commit. It is difficult to see how this is related to the original change and what you are adding/removing here.

Reply:
Is that a requested change, or just some advice?

Reply:
advice :)

Reply:
still in the process of testing this in a production env.

Reply:
Any feedback? This has been sitting for a while.

Reply:
Pinging @nashif for assignee approval before feature freeze (sorry if this is annoying)